src/components/v5/shared/Breadcrumbs/consts.ts

```diff
@@ -0,0 +1,17 @@
+import { type MessageDescriptor } from 'react-intl';
+
+import { supportedExtensionsConfig } from '~constants';
+
+const extensionsBreadcrumbNames = supportedExtensionsConfig.reduce(
+  (names, extensionConfig) => {
+    return {
+      ...names,
+      [extensionConfig.extensionId]: extensionConfig.name,
+    };
+  },
+  {},
+);
+
+export const breadcrumbItemNames: Record<string, MessageDescriptor> = {
+  ...extensionsBreadcrumbNames,
```

Comment:
Maybe it would make sense to actually add all the route constants here and add a custom translation for each possible breadcrumb at least as a `TODO` for now, otherwise we end up with this edge cases

![Screenshot 2024-09-20 at 10 23 42](https://github.com/user-attachments/assets/209527ac-edfc-4486-875b-5bac921c1459)

![Screenshot 2024-09-20 at 10 24 38](https://github.com/user-attachments/assets/70cbf695-d1cd-4d37-86c6-65ae5fd1867d)

Though for these 2 I would suggest to tackle them now 🙏